invenio_vocabularies/services/permissions.py

```diff
@@ -9,17 +9,19 @@
 """Vocabulary permissions."""
 
 from invenio_records_permissions import RecordPermissionPolicy
-from invenio_records_permissions.generators import AnyUser, SystemProcess
+from invenio_records_permissions.generators import SystemProcess
+
+from invenio_vocabularies.services.generators import Tags
 
 
 class PermissionPolicy(RecordPermissionPolicy):
     """Permission policy."""
 
-    can_search = [SystemProcess(), AnyUser()]
-    can_read = [SystemProcess(), AnyUser()]
+    can_search = [SystemProcess(), Tags(exclude=["non-searchable"])]
+    can_read = [SystemProcess(), Tags(exclude=["non-searchable"])]
```

Comment:
Is it really what we want here that nobody at all can search non-searchable names?

Reply:
People with admin permission can search for it. If there is other permissions that should have access to it let me know. But to me it makes sense that only administrators do have.